app/views/admin/logs/_log.html.slim

```diff
@@ -35,10 +37,10 @@
 
     .log-info
       - if log.additional_notes
-        span.bold = t("log.additional_notes")
+        span.bold = t("log.additional_notes", punctuation: ":")
```

Comment:
Not supper happy with passing punctuation. If there is a different way to do this, let me know.